#!/usr/bin/env python
#To launch gui: rosrun rqt_gui rqt_gui -s reconfigure
PACKAGE = "hand_rviz"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pot0", double_t, 0, "The minimum value of potentiometer 0",    500, 0, 1000)
gen.add("pot1", double_t, 0, "The minimum value of potentiometer 1",    500, 0, 1000)
gen.add("pot2", double_t, 0, "The minimum value of potentiometer 2",    500, 0, 1000)
gen.add("pot3", double_t, 0, "The minimum value of potentiometer 3",    500, 0, 1000)
gen.add("pot4", double_t, 0, "The minimum value of potentiometer 4",    500, 0, 1000)
gen.add("pot5", double_t, 0, "The minimum value of potentiometer 5",    500, 0, 1000)


exit(gen.generate(PACKAGE, "hand_rviz", "PotMins"))
